backend/src/utils.ts

```diff
@@ -1,5 +1,6 @@
 import escapeHTML from 'escape-html'
 import { User, Post, Comment } from '.prisma/client'
+import { readTime } from './utils/post'
```

Comment:
What I'm seeing in the diff of this file does not match what is currently on `master`
![image](https://user-images.githubusercontent.com/34203886/86504031-f3a2c280-bd68-11ea-83b6-3a2d3fef62fa.png)